src/app/app-routing.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule }             from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { CarsComponent }      from './cars/cars.component';
+import { CarDetailComponent }  from './car-detail/car-detail.component';
+
+const routes: Routes = [
+  { path: '', redirectTo: '/cars', pathMatch: 'full' },
+  { path: 'detail/:id', component: CarDetailComponent },
```

Comment:
wouldn't make sense `cars/:id` instead of `detail/:id`?

Reply:
Yes, indeed. It makes more sense. Fixed.